src/main/scala/higherkindness/rules_scala/workers/zinc/compile/ZincRunner.scala

```diff
@@ -79,7 +79,7 @@ object ZincRunner extends WorkerMain[Namespace] {
   protected[this] def work(worker: Namespace, args: Array[String], out: PrintStream) = {
     val usePersistence: Boolean = worker.getBoolean("use_persistence") match {
       case p: java.lang.Boolean => p
-      case null                 => true
+      case null                 => false
```

Comment:
> At some point we should tie this option into the ZincConfiguration
parameters in Bazel land.

Anywhere where we can file this? 

Additionally, iirc we're comfortable with disabling this due to how RBE handles actions, where in our current state persistence isn't useful, but after persistent workers we may want persistence, right?
If so, we should also make sure that that is documented somewhere to do.

Reply:
Currently this is just filed in my brain. I started work on this in this PR with
```
if zinc_configuration.incremental:
        execution_requirements_tags["no-sandbox"] = "1"
```

In `phase_zinc_compile.bzl`. I just need to extend it to do more than that.

I plan to keep making improvements, but the PR was already so large I just wanted to get a PR up before making any more changes.